src/components/TodoTemporary.tsx

```diff
@@ -0,0 +1,46 @@
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  temporaryTodo: Todo | null;
+};
+
+export const TodoTemporary = ({ temporaryTodo }: Props) => {
+  if (temporaryTodo !== null) {
```

Comment:
I think this if statement should handle !temporaryTodo then render something when it's not present